### DISCUSSION TITLE
Support for mongodb

### DISCUSSION THREAD

**Discussion by chaitanyabsprip** [kndndrj/nvim-dbee#111] [Q&A]
I have been using mongo compass for mongodb. I wish to switch to nvim-dbee. I am able to connect to the database, however, I'm not sure about how to write queries for mongo with nvim-dbee. I can only find sql examples. What is the format of the queries for mongo using nvim-dbee?

**Comment by dimas-cyriaco** [comment]
I would like to know that too.

**Comment by dimas-cyriaco** [comment]
It seems the filter should be a key in this object. Something like this:

```json
{"find": "users", "filter": { "name": "Michael Jackson" } }
```

**Comment by esatterwhite** [comment]
Has there been a solution to this? The notes always save as `.sql` files and try to execute them as such. I cant seem to run a query outside of the canned `List` query from the dbee UI.

**Comment by esatterwhite** [comment]
@kndndrj could you provide an example of using dbee with mongodb, it isn't very apparent

**Reply by kndndrj** [reply]
Hey, to be honest it's been a long time since I've touched this plugin and even longer that I needed to work with mongo (🙂). But if I remember correctly, the queries should be supplied in a json equivilent to bson - behavior boils down to [this function](https://pkg.go.dev/go.mongodb.org/mongo-driver/bson?utm_source=godoc#UnmarshalExtJSON).

Ergonomics are not really good as I remember though.

**Reply by esatterwhite** [reply]
Right, when you save.a note, it forces a `.sql` extension and when you try to run it, it can't be parsed as sql. 

I don't think it works at all.
